pgroonga/migrations/0002_html_escape_subject.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from django.db import models, migrations
+from django.contrib.postgres import operations
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pgroonga', '0001_enable'),
+    ]
+
+    database_setting = settings.DATABASES["default"]
+    if "postgres" in database_setting["ENGINE"]:
```

Comment:
We only support postgres, so I'm not sure we need this check?

Reply:
Yeah I can remove that, I copied it from the original migration (`pgroonga/migrations/0001_enable.py`)

Reply:
Initially, I moved the code from `zerver/migrations/0001_initial.py`.
The check has been gone in the file, so we can remove the check from this file and `pgroonga/migrations/0001_enable.py`.